uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/BAO/AssignmentTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ *
+ * @group headless
+ */
+class CRM_Tasksassignments_BAO_AssignmentTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
+
+  use TasksassignmentsTestTrait;
+
+  private $contact_id;
+
+  private $case_type_id;
+
+  private $open_status_id;
+
+  private $closed_status_id;
+
+  public function setUpHeadless() {
+    return \Civi\Test::headless()
+      ->installMe(__DIR__)
+      ->apply();
+  }
+
+  public function setUp() {
+    parent::setUp();
```

Comment:
No need for this since your extending PHPUnit_Framework_TestCase, which has an empty setUp implementation.